WORKSPACE

```diff
@@ -34,6 +34,14 @@ http_archive(
     strip_prefix = "googletest-release-1.8.1",
 )
 
+http_archive(
+    name = "linuxsrc",
+    url = "https://cdn.kernel.org/pub/linux/kernel/v4.x/linux-4.15.18.tar.gz",
```

Comment:
How did you choose this particular kernel version? I don't necessary object to using the oldest headers with the features that we need, however we should at least document the selection process in the change description

Reply:
I would actually document the choice in comments just above line 37. Even if it's just "# Most recent kernel running on X at time the code submitted".

We'll need to update the kernel header eventually, and if it was me tasked with the challenge, I'd be like "why did they pick this specific kernel? any specific feature? ...", having some context will help.

Reply:
I have updated with a comment